### ISSUE TITLE
Can't use Dora

### ISSUE THREAD

**Issue by gold90123** [facebookresearch/demucs#459]
![圖片](https://user-images.githubusercontent.com/76434705/228443029-b1cce957-4c4e-4119-ad42-7c1d9a49f513.png)

I just can't use anything about dora, including dora run, dora info or something else.
I tried everything I saw on here, I did

`conda env update -f environment-cuda.yml `
`pip install -r requirements.txt`
`pip install -e.`

I also reinstall dora-search again, but nothing helped.
pls, someone help me

Also, I would like to ask, if it's necessary to utilize dora to train this model, or I can just use the command
`python -m demucs.train`

**Comment by gold90123** [comment]
Hey guys, I switched to a different computer that's running on the Windows operating system, and did the same thing again, and my Dora works!! How strange is this !!
( However, The original system I used was Ubuntu 20.04.4 LTS, which is a computer located in my school lab.)
I'm still in need of some help to get the model run properly on the Ubuntu machine, because the hardware of that machine ismuch powerful than my own computer.
pls help me!!

**Comment by gold90123** [comment]
THANK GOD, I finally solved the problem!
I discovered that I was able to successfully run `from dora import hydra_main` when using the Python interpreter at the command line.
<img width="429" alt="success" src="https://user-images.githubusercontent.com/76434705/229813961-9aa91f75-33c3-4bdd-91e0-b3a6b89280f7.PNG">
This indicates that everything was working correctly, except for the 'dora' command itself.
<img width="733" alt="error" src="https://user-images.githubusercontent.com/76434705/229814095-76750f54-d4fe-45d5-be95-4ea42dd7409c.png">
Upon further investigation, I found that the 'dora' command was using packages from Python3.8, while my demucs environment was running Python 3.9.
To fix this, I first deleted python3.8.
Next, I noticed that the 'dora' command was installed in the command line tool box at ~/.local/bin, so I removed the dora
executable file from that directory.
<img width="928" alt="dora command" src="https://user-images.githubusercontent.com/76434705/229814996-bbaa8d02-114c-49e4-8153-5d7399f093a7.PNG">
Finally, everything works perfectly fine now!!
